test/intersect/bug223_sv1_a.vcf

```diff
@@ -0,0 +1,3 @@
+##fileformat=VCFv4.1
+chr1	1	a	G	<DEL>	70.90	.	TOOL=LUMPY;SVTYPE=DEL;SVLEN=-389,-4611;END=253195;STR=+-:4;IMPRECISE;CIPOS=-2,137;CIEND=0,0;EVENT=791255;SUP=4;PESUP=4;SRSUP=0;EV=PE;PRIN;CSQ=intergenic_variant||||||||||
+chr1	4	a	G	<INS>	70.90	.	TOOL=LUMPY;SVTYPE=DEL;SVLEN=-4611;END=253195;STR=+-:4;IMPRECISE;CIPOS=-2,137;CIEND=0,0;EVENT=791255;SUP=4;PESUP=4;SRSUP=0;EV=PE;PRIN;CSQ=intergenic_variant||||||||||
```

Comment:
@nkindlon I was referring to the INFO column here. Although the ALT field has `<INS>`, the INFO field has `SVTYPE=DEL` which can be confusing.